app/helpers/standard_form_builder.rb

```diff
@@ -75,6 +75,7 @@ def rich_text_area(attr, html_options = {})
   def number_field(attr, html_options = {})
     html_options[:size] ||= 10
     html_options[:class] ||= 'span2'
+    html_options[:pattern] ||= '([0-9]*[.])?[0-9]+'
```

Comment:
Negative Zahlen schliesst du damit neu aus, ebenso scientific notation falls das ein Float-Input ist, und wahrscheinlich noch mehr Dinge die ich gerade vergesse. Warum ist das neu nötig? Sollten wir nicht besser mit Validierungen oder mit `<input type="number">` arbeiten?

Reply:
Alles berechtigte Anmerkungen merci
Mein Grund nicht `type="number"` zu verwenden ist, dass auch das zumindest in erster Linie nur mit Integer funktioniert (vielleicht hab ich da aber auch meine Suche zu früh abgebrochen). Zusätzlich bekommen wir dadurch einen ziemlich hässlichen Spinner im Inputfeld, welches wir dann per CSS wieder entfernen müssen (zumindest laut meiner Einschätzung, vielleicht will jemand das Ding ja).
Wenn es aber die anderen Notationen besser unterstützt und Kommazahlen kein Problem sind, können wir das gerne anpassen :)

Reply:
Ich finde, dass das `:pattern` schon Bände spricht. Helper-Test sehe ich hier als nicht nötig an und schliesse mich damit @TheWalkingLeek an :-)